src/tokens/solana.tokenlist.json

```diff
@@ -11288,11 +11288,30 @@
       "extensions": {
         "website": "https://www.mercurial.finance/"
       }
+    },
+    {
+      "chainId": 101,
+      "address": "4JmyYaduPQPVQqEAS2MG8K5sG7EwKHvPAFHDFjtUUsSx",
```

Comment:
remade the token contract
4JmyYaduPQPVQqEAS2MG8K5sG7EwKHvPAFHDFjtUUsSx
removed third party wallet holders